File: 3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.py
class Solution:
    def missingMultiple(self, nums: List[int], k: int) -> int:
        freq=[0]*10005
        for i in range(len(nums)):
            freq[nums[i]]+=1
        j=1
        while True:
            prod=k*j
            if(freq[prod]==0):
                return prod
            j+=1

File: 3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.java
class Solution {
    public int missingMultiple(int[] nums, int k) {
        int[] freq=new int[10005];
        for(int i=0;i<nums.length;i++)
        {
            freq[nums[i]]++;
        }
        for(int j=1; ;j++)
        {
            int prod=k*j;
            if(freq[prod]==0)
            {
                return prod;
            }
        }
    }
}